Include/internal/pycore_long.h

```diff
@@ -78,6 +78,14 @@ static inline PyObject* _PyLong_FromUnsignedChar(unsigned char i)
     return Py_NewRef((PyObject *)&_PyLong_SMALL_INTS[_PY_NSMALLNEGINTS+i]);
 }
 
+/* Return 1 if the argument is negative or a multi-digit int */
+static inline int
+_PyLong_Negative_or_multi_digit_int(PyObject* sub) {
```

Comment:
Please revert this change since it's now discussed in a separated PR: PR #100064

Reply:
Done!